src/Storage/Storage.Management/ChangeLog.md

```diff
@@ -18,6 +18,8 @@
         - Additional information about change #1
 -->
 ## Upcoming Release
+* Supported Planned type in Storage account failover type 
```

Comment:
Please follow this code change to do soft failover feature change.
This code change is agreed with feature team before.
https://github.com/wastoresh/azure-powershell/commit/bb36ed0d57021c214d06c194c361822e0cce9868#diff-c86d593e2a3d6b8754aaaaaefb88480871821afddc56e4e287854b3d05893704